.github/workflows/docker-builder-publisher.yaml

```diff
@@ -37,7 +41,7 @@ jobs:
       - name: Azure login
         uses: azure/login@v1
         with:
-          client-id: 9c8d1e31-54b5-4156-b1d0-408135e2b55b
+          client-id: 7e26ba04-3da0-4241-b517-c12b8205af8a
```

Comment:
I have not checked that ID, but it should be the one you have terraformed federation for.